packages/react-day-picker/src/components/Head/utils/getWeekdays.ts

```diff
@@ -12,7 +12,10 @@ export function getWeekdays(
   /** The index of the first day of the week (0 - Sunday) */
   weekStartsOn?: 0 | 1 | 2 | 3 | 4 | 5 | 6
 ): Date[] {
-  const start = startOfWeek(new Date(), { locale, weekStartsOn });
+  const start = startOfWeek(new Date(), {
+    ...(locale && { locale }),
+    ...(weekStartsOn && { weekStartsOn })
+  });
```

Comment:
This change is weird :) I wanted to keep for `locale` and `weekStartsOn` the same type as the date-fns functions.

Are there reasons why you changed these props too? Was the compiler filing while setting the config?

Reply:
Yes. The `date-fns` `startOfWeek` function do not accept `locale` and `weekStartsOn` as `undefined`.